tools/envoy_headersplit/README.md

```diff
@@ -0,0 +1,13 @@
+# Envoy Header Split
+Tool for spliting monolith header files in envoy to speed up compilation
+
+
+Steps to divide Envoy mock headers:
+
+1. run `headersplit.py` to divide the monolith mock header into different classes
+
+2. Resolve bazel dependency for the divided classes manually and remove unused includes for them
+
+3. run `replace_includes.py` to replace superfuluous #includes in Envoy directory after dividing. it will also modify the corresponding Bazel `BUILD` file.
```

Comment:
An example command might be useful here; I'm not sure what you mean by "Envoy directory"

Spelling: "superfluous"

Reply:
fixed